error TS2504: Type 'AsyncIterator<Plan, any, undefined>' must have a '[Symbol.asyncIterator]()' method that returns an async iterator.

**Describe the bug**

Upgrading to recurly Node.js client v3.3.0 (released yesterday Dec 19) causes TypeScript v3.7.4 build failure with this code:

```ts
  for await (const plan of client.listPlans().each()) {
    plans.push(plan);
  }
```

```
error TS2504: Type 'AsyncIterator<Plan, any, undefined>' must have a '[Symbol.asyncIterator]()' method that returns an async iterator.
```

--



**To Reproduce**

Steps to reproduce the behavior.

If possible, please provide example code which we can run to reproduce the bug.
The faster we can reproduce the bug, the faster we can help you.

**Expected behavior**

A clear and concise description of what you expected to happen.

**Your Environment**

- Which version of this library are you using? 3.3.0
- Which version of the language are you using? Node 12, TypeScript 3.7.4

--- a/lib/recurly.d.ts
+++ b/lib/recurly.d.ts
@@ -2087,8 +2087,8 @@ export interface Empty {
 export declare class Pager<T> {
   count(): number;
   first(): T;
-  each(): AsyncIterator<T>;
-  eachPage(): AsyncIterator<T>;
+  each(): AsyncIterable<T>;
+  eachPage(): AsyncIterable<T>;
 }
 
 export declare class Client {

